src/crewai/utilities/events/event_listener.py

```diff
@@ -370,6 +374,30 @@ def on_llm_stream_chunk(source, event: LLMStreamChunkEvent):
             print(content, end="", flush=True)
             self.next_chunk = self.text_stream.tell()
 
+        # ----------- LLM GUARDRAIL EVENTS -----------
+
+        @crewai_event_bus.on(LLMGuardrailStartedEvent)
+        def on_llm_guardrail_started(source, event: LLMGuardrailStartedEvent):
+            guardrail_name = "Guardrail"
+            if isinstance(event.guardrail, str):
+                guardrail_name = (
+                    event.guardrail[:50] + "..."
+                    if len(event.guardrail) > 50
+                    else event.guardrail
+                )
+            elif hasattr(event.guardrail, "__name__"):
+                guardrail_name = event.guardrail.__name__
+            elif hasattr(event.guardrail, "__class__"):
+                guardrail_name = event.guardrail.__class__.__name__
```

Comment:
We [make sure](https://github.com/crewAIInc/crewAI/blob/main/src/crewai/utilities/events/llm_guardrail_events.py#L27) the `self.guardrail` is always a string. Right? 

So I wonder to see when/how the elif condition will be fired. Can you record a video showing that?